app/Resources/views/customer/product/details.html.twig

```diff
@@ -0,0 +1,14 @@
+{% extends ':customer:base.html.twig' %}
+
+{% block body %}
+    <div class="row">
+        <div class="col-lg-12">
+            <h1>{{ product.metadata.name }}</h1>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-lg-4">
+            <h4>Price: {{ product.price|number_format(2, '.', ',') }} {{ product.currency }}</h4>
```

Comment:
afaik there is no such filter in twig by default and I also don't know any extension that would give us it.